### PR TITLE
Feature/laravel 8 upgrade

### PR DESCRIPTION
**Pull request by bendeeson**
Updates to opendialog app to support laravel 8.
* Changes CORS library from `spatie/laravel-cors` to `fruitcake/laravel-cors`. Also includes a published `cors.php` config file that enables cors for all paths, but allows access from anywhere.
* Updated mailer config file.

### CODE REVIEW

File: app/Http/Resources/ConversationCollection.php [hunk 1]
```diff
@@ -15,8 +15,8 @@ class ConversationCollection extends ResourceCollection
      */
     public function toArray($request)
     {
-        $this->collection->transform(function (Conversation $c) {
-            return new ConversationBuilderResource($c);
+        $this->collection->transform(function (ConversationResource $c) {
```
Comment (bendeeson) [inline app/Http/Resources/ConversationCollection.php:18]: See https://github.com/laravel/framework/issues/35906

File: composer.json [hunk 2]
```diff
@@ -9,28 +9,30 @@
...
-        "laravel/tinker": "^1.0",
+        "fruitcake/laravel-cors": "^2.0",
+        "laravel/framework": "^8.0",
+        "laravel/legacy-factories": "^1.1",
+        "laravel/tinker": "^2.0",
+        "laravel/ui": "^3.0",
         "maennchen/zipstream-php": "^2.0",
-        "opendialogai/core": "0.7.x-dev",
-        "opendialogai/webchat": "0.7.x-dev",
+        "opendialogai/core": "dev-feature/laravel-8-upgrade",
```
Comment (bendeeson) [inline composer.json:18]: Reminder to replace these with 0.7x-dev, once core/webchat are merged.

File: composer.json [hunk 3]
```diff
@@ -9,28 +9,30 @@
...
+        "opendialogai/webchat": "dev-feature/laravel-8-upgrade",
         "phalcongelist/php-diff": "^2.0",
         "predis/predis": "^1.1",
         "propaganistas/laravel-phone": "^4.2",
         "spatie/commonmark-highlighter": "^2.0",
-        "spatie/laravel-cors": "^1.6",
-        "spatie/laravel-flash": "1.3",
-        "srmklive/authy": "^0.6"
+        "spatie/laravel-flash": "^1.3",
+        "srmklive/authy": "^0.7.0"
```
Comment (bendeeson) [inline composer.json:25]: srmklive/authy provides two-factor authentication, but doesn't seem to have been installed correctly (Service provider is not registered in `config/app`) in 0.7.x. Is this used at all?

File: app/Console/Commands/Specification/ExportConversations.php [hunk 4]
```diff
@@ -16,7 +16,7 @@ public function handle()
     {
         $conversationName = $this->argument('conversation');
 
-        if ($this->option('yes') || $this->option('y')) {
+        if ($this->option('yes')) {
```
Comment (stu-greenshoots) [inline app/Console/Commands/Specification/ExportConversations.php:19]: 👍

File: composer.json (hunk 3 above)
Comment (stu-greenshoots) [reply composer.json:25]: I just tested the 2FA flow on this branch and it's still working, so I think we're good 👍

**Review by stu-greenshoots — APPROVED** [review]
Looks great @bendeeson  - again, happy to merge one the compser branches have been updated.

I have tested the 2FA flow and sending of emails, all working as expected 👍

**Pull request merged**